webui/react/package.json

```diff
@@ -138,6 +138,7 @@
     "jsdom": "^16.7.0",
     "morgan": "^1.10.0",
     "npm-force-resolutions": "0.0.10",
+    "playwright-page-model-base": "npm:@hpe.com/playwright-page-model-base@^0.2.5",
```

Comment:
npm i playwright-page-model-base@npm:@hpe.com/playwright-page-model-base --save-dev

Reply:
we do this for hew because for a time we were installing directly from github -- it might be confusing to introduce it in new code.

Reply:
I'll make sure to leave a reference to this command in the package readme. i like how i looks compared to the @hpe on every import